cmd/monaco/integrationtest/v2/all_configs_integration_test.go

```diff
@@ -51,20 +50,12 @@ func runAllConfigsTest(t *testing.T, specificEnvironment string) {
 	}
 
 	RunIntegrationWithCleanupGivenEnvs(t, configFolder, manifest, specificEnvironment, "AllConfigs", envVars, func(fs afero.Fs, _ TestContext) {
-
 		// This causes a POST for all configs:
-
-		cmd := runner.BuildCli(fs)
-		cmd.SetArgs([]string{"deploy", "--verbose", manifest, "--environment", specificEnvironment})
-		err := cmd.Execute()
-
+		err := monaco.RunWithFsf(fs, "monaco deploy %s --environment=%s --verbose", manifest, specificEnvironment)
```

Comment:
Nit: maybe lets rename this function to avoid the question what a "Fsf" is :D
```
RunfWithFS(...)
```

Reply:
Can it be `RunWithFSf`?
There is some convention for this type of formatting that a function ends with `f`. Intellij recognizes this (check if enough arguments exist).

Reply:
ok :)